src/users/users.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { UsersService } from './users.service';
+import { UsersController } from './users.controller';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { User } from '../model/user.entity';
+
+@Module({
+  // app.module has root connection, but for each database must import 'TypeOrmModule.forFeature([entityName])'
+  imports: [TypeOrmModule.forFeature([User])],
```

Comment:
Read [here](https://docs.nestjs.com/techniques/database#repository-pattern) to learn more about `forRoot()` and `forFeature()`

Reply:
after discussion, keep the original version